frontend/app-development/layout/SettingsModalButton/SettingsModal/components/Tabs/AccessControlTab/AccessControlTab.tsx

```diff
@@ -31,6 +46,8 @@ export type AccessControlTabProps = {
  */
 export const AccessControlTab = ({ org, app }: AccessControlTabProps): ReactNode => {
   const { t } = useTranslation();
+  const [checkedCheckboxes, setCheckedCheckboxes] = useState<string[]>([]);
```

Comment:
Do we need a state for this? Can we only use the partyTypes fetched from appmetadata to visaulize what checkboxes are checked? Everytime a checkbox is clicked we call the handleChange method which updates the appmetadata so it will always reflect what should be shown. Except if none are check which will result in checking the `All types` but that is doable without this state is suppose 🫣

Reply:
Then we will also get passed the issue in the tests that sets all checkboxes to unchecked as initial value until something is clicked